### PR TITLE
Add documentation for rate limiting rules

### PR DESCRIPTION
**Pull request by HeJones**
### What

- Add documentation to the developer site. to explain the rate limiting rules that have been applied to the API

### How to review

- Check that there are no typos and that the documentation reads well and is clear. 

### Who can review

Anyone

### CODE REVIEW

File: static/index.md [hunk 1]
```diff
@@ -33,6 +33,15 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
+The following rate limiting rules have been implemented on the API
```
Comment (sandypadmanabhan) [inline static/index.md:37]: Few lines to explain why we are adding the rate limit would be good. something along the lines of "We limit the number of requests you can make to dp API's to protect our service against excessive use and denial of service attacks ."

Comment (sandypadmanabhan) [reply static/index.md:37]: We also need to mention about the  429 response when we exceed the limit stated and about the Retry-header as mentioned in the trello ticket

Comment (sandypadmanabhan) [reply static/index.md:37]: And can you mention both the domains as well

File: static/index.md [hunk 2]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
...
+
+* 200 requests per 1 minute
+
+They have been applied to following domains. 
+
+* api.ons.gov.uk
+
+* api.beta.ons.gov.uk
+
+If requests are made that reach these limits the IP address of the client will be blocked for 1 minute. The client will receive a 429 response, the retry-header contains the seconds remaining until a retry can be made.
```
Comment (sandypadmanabhan) [inline static/index.md:52]: the retry-header `shows` or something similar would make sense . `retry-header contains the seconds' seems out of place .

File: static/index.md [hunk 3]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
+
+Rate limiting rules have been implemented to limit the number of requests a client can make to The Office for National Statistics API. This is to protect the service against excessive use and denial of service attacks. 
+
+The following rate limiting rules have been implemented on the API.
```
Comment (gedge) [inline static/index.md:40]:
```suggestion
The following rate limits have been implemented on the API:
```

File: static/index.md [hunk 4]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
...
+
+Rate limiting rules have been implemented to limit the number of requests a client can make to The Office for National Statistics API. This is to protect the service against excessive use and denial of service attacks. 
+
+The following rate limiting rules have been implemented on the API. 
+
+* 120 requests per 10 seconds
+
+* 200 requests per 1 minute
+
+They have been applied to following domains.
```
Comment (gedge) [inline static/index.md:46]:
```suggestion
They have been applied to following domains:
```

File: static/index.md [hunk 5]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
+
+Rate limiting rules have been implemented to limit the number of requests a client can make to The Office for National Statistics API. This is to protect the service against excessive use and denial of service attacks.
```
Comment (gedge) [inline static/index.md:38]:
```suggestion
Rate limiting has been implemented to contain the number of requests a client can make to our API. This is to protect the service against excessive use and denial of service attacks. 
```

1. repeated 'limit' was repetitive 😄 
2. removing 'rules' sounds more direct

**Review by gedge — CHANGES_REQUESTED** [review]

File: static/index.md [hunk 6]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
...
+
+* 200 requests per 1 minute
+
+They have been applied to following domains: 
+
+* api.ons.gov.uk
+
+* api.beta.ons.gov.uk
+
+If requests are made that reach these limits the IP address of the client will be blocked for 1 minute. The client will receive a JSON response with a 429 error code, the retry-after header shows the seconds remaining until a retry can be made.
```
Comment (sandypadmanabhan) [inline static/index.md:52]: The client will receive a JSON response with a 429 error code. The Retry-After header shows the seconds remaining until a retry can be made.

File: static/index.md [hunk 7]
```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
```
Comment (sandypadmanabhan) [inline static/index.md:36]: Looking at other subheading , this should be `Rate limiting rules`. L and R (lowercase)

**Review by sandypadmanabhan — APPROVED** [review]

**Review by gedge — APPROVED** [review]

**Pull request merged**